backend/prisma/README.md

```diff
@@ -4,3 +4,6 @@ They have to be migrated in the container as follows:
 docker exec -it backend bash
 npx prisma migrate dev --name <name migration>
 exit
+
+To open Prisma Studio:
+npx prisma studio
```

Comment:
Also need to put the ".env" under "backend" directory and change the DB_CONTAINER variable to "localhost". (That's how I use prisma studio :D )